packages/helm/symphony/templates/redis.yaml

```diff
@@ -1,3 +1,18 @@
+{{- if and .Values.redis.enabled .Values.redis.persistentVolume.enabled}}
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: redis-pvc
+  namespace: {{ .Release.Namespace }}
+spec:
+  storageClassName: {{ .Values.redis.persistentVolume.storageclass }}
```

Comment:
This requires a PV already exists. How can we help users to set it up?

Reply:
This only requires storage class to be present. And the PV will be dynamically provisioned from storage class with name prefix of PVC. The lifetime of generated PV will be the same as PVC.